src/darker/tests/test_main.py

```diff
@@ -929,3 +937,22 @@ def test_stdout_path_resolution(git_repo, capsys):
 
     assert result == 0
     assert capsys.readouterr().out == 'print("foo")\n'
+
+
+@pytest.mark.skipif(not sys.platform.startswith("win"), reason="Argument length limit on Windows")
+def test_large_file_count(git_repo):
+    # For PR #542 - large character count for changed files
+    # on windows breaks subprocess
+    # Need to exceed 32762 characters
+    files = {}
+
+    for _d in range(0, 210):
+        path = "src"
+        for _f in range(0, 4):
+            # Of course windows limits the path length too
+            path = f"{path}/{randomword(30)}"
+        files[f"{path}/{randomword(30)}.py"] = randomword(10)
```

Comment:
If you first create just one deep directory structure and then put all the 210 files inside the same directory, you'll only need $4 + 210 = 214$ file operations instead of $210 * 4 = 840$ file operations.